packages/headless/src/commerce.index.ts

```diff
@@ -92,20 +92,14 @@ export {
   SortDirection,
```

Comment:
I haven't yet worked on the Barca Sports integration, so I might end up exporting more (but probably not less). I'm starting with what I believe will be the bare minimum.